drivers/hmis/app/graphql/types/forms/pick_list_option.rb

```diff
@@ -394,6 +396,15 @@ def self.open_hoh_enrollments_for_project(project)
       end
     end
 
+    def self.external_form_types_for_project(project)
+      return [] unless project.present?
+
+      Hmis::Form::Instance.for_project(project).
+        preload(:definition).
+        order(:id).
+        map(&:to_pick_list_option)
+    end
```

Comment:
Whoops, missed this the first time around. I'm seeing 2 issues:
1) this is returning non-external-forms, if there are other forms that have rules for this project (fixed by `with_role`)
2) if for some reason you had two duplicate rules for the project, it would return duplicates(fixed by `uniq`)

```suggestion
      Hmis::Form::Instance.for_project(project).
        with_role(:EXTERNAL_FORM).
        preload(:definition).
        order(:id).
        map(&:to_pick_list_option).uniq
    end
```

Reply:
thank you for the good catch @gigxz !